api/src/pcapi/routes/backoffice/gdpr_user_extract/blueprint.py

```diff
@@ -0,0 +1,51 @@
+from flask import render_template
+import sqlalchemy as sa
+
+from pcapi.core.permissions import models as perm_models
+from pcapi.core.users import models as users_models
+from pcapi.models.feature import FeatureToggle
+from pcapi.routes.backoffice import utils
+
+
+gdpr_extract_blueprint = utils.child_backoffice_blueprint(
+    "gdpr_extract",
+    __name__,
+    url_prefix="/gdpr-extract",
+    permission=perm_models.Permissions.EXTRACT_PUBLIC_ACCOUNT,
+)
+
+
+def _get_gdpr_data() -> list[users_models.GdprUserDataExtract]:
+
+    query = (
+        users_models.GdprUserDataExtract.query.options(
+            sa.orm.joinedload(users_models.GdprUserDataExtract.user).load_only(
+                users_models.User.firstName,
+                users_models.User.lastName,
```

Comment:
```suggestion
            sa.orm.joinedload(users_models.GdprUserDataExtract.user).load_only(
                users_models.User.firstName,
                users_models.User.lastName,
                users_models.User.id,
```
tu as besoin de l'id pour le lien dans le BO (`build_public_user_name_to_details_link` )

Reply:
Je pense que l'ORM récupère toujours l'`id`, donc implicitement ça marche (sinon l'ORM ne sait pas sur quel `id` faire la "N+1").

Reply:
Je l'ajoute quand meme